woocommerce/wc-admin
# Add/7481 cmd to collect testing instructions

Fixes #7481 

This PR adds a new console command to retrieve test instructions from PR #s in a given version from the changelog.txt and write them into `TESTING-INSTRUCTIONs.md`

- We no longer need to write test instructions into `TESTING-INSTRUCTIONS.md` manually.
- No more merge conflicts with `TESTING-INSTRUCTIONS.md`

### Detailed test instructions:

Follow [Prerequisites](https://github.com/woocommerce/woocommerce-admin/blob/add/7481-cmd-to-collect-testing-instructions/bin/test-instruction-logger/README.md) section in the README.md to set Github credentials.


1. Find a version in `changelog.txt`
2. Run `npm run test-instruction-logger write :version --save-to=test.md`
3. Open `test.md` and verify the change.
4. Find a PR that is in the version from the first step and add `no release testing instructions` label.
5. Run `npm run test-instruction-logger write :version --save-to=test.md` again. 
6. Open `test.md` and confirm the PR is not included for this time.

no changelog needed
